src/qt/publishdialog.cpp

```diff
@@ -158,6 +158,21 @@ void PublishDialog::on_button_OkCancel_accepted()
         case 1: SP->m_dw_qos.durability().kind = eprosima::fastdds::dds::TRANSIENT_LOCAL_DURABILITY_QOS; break;
     }
 
+    std::cout<<"Create Shape publisher"<<std::endl;
+    //DATA REPRESENTATION
+    switch (this->ui->comboBox_representation->currentIndex())
```

Comment:
If using brakets per case, we tend to include the `break` statement into the brackets.
Also, start in the next line. You can take the next `switch` statement in the `OWNERSHIP QOS` as a reference